grpcbp/tracing.go

```diff
@@ -44,3 +44,13 @@ func methodSlug(method string) string {
 	split := strings.Split(method, "/")
 	return split[len(split)-1]
 }
+
+// serviceAndMethodSlug splits the UnaryServerInfo.FullMethod and returns
+// the package.service part separate from the method part.
+// ref: https://pkg.go.dev/google.golang.org/grpc#UnaryServerInfo
+func serviceAndMethodSlug(fullMethod string) (string, string) {
+	split := strings.Split(fullMethod, "/")
```

Comment:
strings.SplitN(fullMethod, 3)

Reply:
just in case somehow we're give "/foo/bar/baz" -- so that we return "bar/baz"

You could also trim a / prefix from fullMethod and then SplitN with 2, so that "foo/bar" also works just as "/foo/bar" would